test/geometry/liegroup/test_so3.py

```diff
@@ -79,7 +79,7 @@ def test_unit_norm(self, device, dtype, batch_size):
         s6 = s3.inverse()
         for i in range(batch_size):
             self.assert_close(s1[i].q.norm(), torch.Tensor([[1.0]]))
-            self.assert_close(s2[i].q.norm(), torch.Tensor([[1.0]]))
+            self.assert_close(s2[i].q.norm(), torch.tensor([[1.0]], device=device, dtype=dtype))
             self.assert_close(s3[i].q.norm(), torch.Tensor([[1.0]]))
```

Comment:
Same for the one above, unless you want to declare a shared variable. Pytorch assert close is quite strict about tensor shapes, device and dtypes